Generals/Code/GameEngine/Source/GameLogic/AI/AIStates.cpp

```diff
@@ -4028,8 +4028,11 @@ StateReturnType AIFollowWaypointPathState::update()
 		if (getAdjustsDestination() && ai->isDoingGroundMovement()) {
 			if (!TheAI->pathfinder()->adjustDestination(obj, ai->getLocomotorSet(), &m_goalPosition)) {
 				if (m_currentWaypoint) {
+					// This log causes mismatch on Release (TheSuperHackers @info helmutbuhler 05/05/2025)
```

Comment:
Better move this behind the #ifdef like the other places. And fix the comment format according to our comment style: `TheSuperHackers @info Log causes mismatch in Release`. Several times in this change.